internal/accounts_test.go

```diff
@@ -0,0 +1,177 @@
+package internal
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/algorandfoundation/hack-tui/api"
+	"github.com/oapi-codegen/oapi-codegen/v2/pkg/securityprovider"
+	"github.com/stretchr/testify/assert"
+)
+
+func getAddressesFromGenesis(t *testing.T) ([]string, string, string) {
```

Comment:
This should probably live in `accounts.go` since it's largely application code. Then we can import it in other tests that require accounts